### ISSUE TITLE
Add Example to Measure Performance of Quantized Model

### ISSUE THREAD

**Issue by ecyht2** [mmwillet/TTS.cpp#61]
There should be some way of measuring performance of the initial FP32/FP16 models compared to the quantized version just like the perplexity tool in llama.cpp.

**Comment by mmwillet** [comment]
If I recall correctly, the perplexity tool that your are referring to simply compares the perplexity of the softmax of the models' logits. This won't be applicable for all TTS models as many do not generate via sampling autoregressive logits (e.g. Kokoro and DAC, both for which we should eventually support quantization), but there is no reason that this cannot be applied to Parler and Dia.

Separately, I suspect that, in the long run, we can do better than perplexity for the purposes of determining model consistency. I say this because when I initially looked at the perplexity of quantized Parler models last year I found little drop off but substantial changes in observed performance (especially at Q4).
